hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/PartitionValueExtractorTest.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.hudi.hive;
+
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PartitionValueExtractorTest {
+  @Test
+  public void testHourPartition() {
```

Comment:
IMO, we need to add more test cases to improve the test coverage. wdyt?

Reply:
e.g. some bad cases

Reply:
ok